pkg/core/state/notification_event.go

```diff
@@ -120,6 +130,9 @@ func (aer *AppExecResult) DecodeBinary(r *io.BinReader) {
 	aer.Stack = arr
 	r.ReadArray(&aer.Events)
 	aer.FaultException = r.ReadString()
+	if aer.VMState&0x80 != 0 {
```

Comment:
And reuse this constant here.

Reply:
:eyes: 
https://github.com/nspcc-dev/neo-go/blob/673b26fdb1097e09044776bcf46ad73627610764/pkg/encoding/bigint/bigint.go#L31

https://github.com/nspcc-dev/neo-go/blob/673b26fdb1097e09044776bcf46ad73627610764/pkg/vm/emit/emit.go#L57

Reply:
I'm not a fan of this solution, it's harder to maintain. And I treat the case of VM state as kind of special, because originally it's not designed to store additional information in the MSB (and strictly speaking, it can even be any other free bit).

Reply:
VM and bigint cases are about negative/MSB, it's more obvious from the purpose of this code (especially in the bigint case). This 0x80 however is pure magic and it can be 0x20 or whatever else. Compatibility kludge in its best, we need to remember it. If we're to ever change the DB scheme this will be removed for sure.